docs/src/pages/components/tabs/examples/ControlledTabExample.tsx

```diff
@@ -0,0 +1,16 @@
+import { useState } from 'react';
+import { Tabs, TabItem, Button } from '@aws-amplify/ui-react';
+
+export const ControlledTabExample = () => {
+  const [index, setIndex] = useState(0);
+  return (
+    <Tabs currentIndex={index} onChange={(i: number) => setIndex(i)}>
+      <TabItem title="First">Content of the first tab</TabItem>
+      <TabItem title="Second">
+        The content of the second tab is initially shown because we passed in
+        index 1 to defaultIndex (notice that the tabs are zero-indexed).
+        <Button onClick={() => setIndex(0)}>Go to first tab</Button>
```

Comment:
Perhaps we can set the Button's display as block instead of inline?

Reply:
Yeah that looks better, updated!